cumulus/parachains/runtimes/assets/asset-hub-westend/src/lib.rs

```diff
@@ -924,17 +927,35 @@ impl pallet_xcm_bridge_hub_router::Config<ToRococoXcmRouterInstance> for Runtime
 	type RuntimeEvent = RuntimeEvent;
 	type WeightInfo = weights::pallet_xcm_bridge_hub_router::WeightInfo<Runtime>;
 
-	type UniversalLocation = xcm_config::UniversalLocation;
-	type SiblingBridgeHubLocation = xcm_config::bridging::SiblingBridgeHub;
-	type BridgedNetworkId = xcm_config::bridging::to_rococo::RococoNetwork;
-	type Bridges = xcm_config::bridging::NetworkExportTable;
 	type DestinationVersion = PolkadotXcm;
 
-	type ToBridgeHubSender = XcmpQueue;
-	type LocalXcmChannelManager =
-		cumulus_pallet_xcmp_queue::bridging::InAndOutXcmpChannelStatusProvider<Runtime>;
+	// Let's use `SovereignPaidRemoteExporter`, which sends `ExportMessage` over HRMP to the sibling
+	// BridgeHub.
+	type ToBridgeHubSender = SovereignPaidRemoteExporter<
+		// `ExporterFor` wrapper handling dynamic fees for congestion.
+		pallet_xcm_bridge_hub_router::impls::ViaRemoteBridgeHubExporter<
+			Runtime,
+			ToRococoXcmRouterInstance,
+			NetworkExportTable<xcm_config::bridging::to_rococo::BridgeTable>,
+			xcm_config::bridging::to_rococo::RococoNetwork,
+			xcm_config::bridging::SiblingBridgeHub,
+		>,
+		XcmpQueue,
+		xcm_config::UniversalLocation,
+	>;
+
+	// For congestion - resolves `BridgeId` using the same algorithm as `pallet_xcm_bridge_hub` on
+	// the BH.
+	type BridgeIdResolver = pallet_xcm_bridge_hub_router::impls::EnsureIsRemoteBridgeIdResolver<
+		xcm_config::UniversalLocation,
+	>;
+	// For congestion - allow only calls from BH.
+	type BridgeHubOrigin =
+		AsEnsureOriginWithArg<EnsureXcm<Equals<xcm_config::bridging::SiblingBridgeHub>>>;
```

Comment:
Seems param `bridge_id` is not used when check the origin here, would be simpler to just use `EnsureXcm<Equals<xcm_config::bridging::SiblingBridgeHub>>>`?